src/components/app-rating/AppRating.jsx

```diff
@@ -0,0 +1,59 @@
+import { useTranslation } from 'react-i18next'
+import StarSharp from '@mui/icons-material/StarSharp'
+import Box from '@mui/material/Box'
+import Rating from '@mui/material/Rating'
+import Typography from '@mui/material/Typography'
+
+import { styles } from '~/components/app-rating/AppRating.styles'
+
+const starsSize = { small: 'body2', medium: 'body1', large: 'h6' }
+
+const AppRating = ({ value = '', numberVariant , size = 'small', withBackground, mobile, reviews, spacing = 1, ...props }) => {
+  const { t } = useTranslation()
+
+  const optionalStyles = {
+    backgroundColor: withBackground && 'primary.50',
+    flexDirection: numberVariant === 'big' && 'column'
+  }
+
+  const smallnumberVariant = numberVariant === 'small' && (
```

Comment:
We can just assign the template, and then use ternary operator.

Reply:
that value can be undefined, so i change to two variables